frontend/components/AddSite/TemplateSiteList/index.js

```diff
@@ -53,35 +30,24 @@ export class TemplateList extends React.Component {
   render() {
     const { templates } = this.props;
 
-    const templateKeys = Object.keys(templates);
-    // if there are fewer templates than cells per row,
-    // fill the space with them
-    const cellsPerRow = Math.min(templateKeys.length, MAX_CELLS_PER_ROW);
-    // generate a two-dimensional array of template keys
-    const templateRows = createRowsOf(templateKeys, cellsPerRow);
-
-    let index = 0;
-
-    const templateGrid = templateRows.map(row => (
-      <div className="federalist-template-list">
-        {row.map((templateName) => {
-          const template = templates[templateName];
-          return (
-            <TemplateSite
-              key={templateName}
-              name={templateName}
-              index={index++} // eslint-disable-line no-plusplus
-              thumb={template.thumb}
-              active={this.state.activeChildId}
-              handleChooseActive={this.handleChooseActive}
-              handleSubmit={this.props.handleSubmitTemplate}
-              defaultOwner={this.props.defaultOwner}
-              {...template}
-            />
-          );
-        })}
-      </div>
-    ));
+    const templateGrid = Object.keys(templates).map((templateName, index) => {
+      const template = templates[templateName];
+
+      return (
+        <div className="federalist-template-list" key={templateName}>
+          <TemplateSite
+            name={template.title}
+            index={index} // eslint-disable-line no-plusplus
```

Comment:
can get rid of this `// eslint` comment as well

Reply:
oh yeah duh